sprint0/sprint0/Input/Command/FoodCommand.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Author: Stuti Shah
+namespace sprint0
+{
+    class FoodCommand : ICommand
+    {
+        private readonly Game1 game;
+        private HUDManager HUD;
+        private PlayerItems item = PlayerItems.Food;
+        private int health = 5;
+
+        public FoodCommand(Game1 game) => this.game = game;
```

Comment:
does the functionality of FoodCommand and execute have to be separated, especially given that FoodCommand only defines game?